buoy1: [5, 0]
buoy2: [5, 5]
buoy3: [0, 0]
